src/app/api/getUserData.ts

```
@@ -0,0 +1,38 @@
+		{
+			method: 'GET',
+			headers: {
+				'Content-Type': 'application/json',
+				Authorization: `Bearer ${token}`,
+			},
+		},
+	);
+
+	if (!response.ok) {
+		const error = await response.json();
+		console.error(error);
```

ESLint

[error] Unexpected console statement.

(no-console)